### ISSUE TITLE
TypeError: Skyweb is not a function

### ISSUE THREAD

**Issue by findel** [ShyykoSerhiy/skyweb#47]
I've just updated to 0.1.1, and maybe I'm missing something?

Using the following code:

```
Skyweb = require('skyweb');
var skyweb = new Skyweb();
```

I'm getting the following error:

```
var skyweb = new Skyweb();
             ^
TypeError: Skyweb is not a function
```

**Comment by ShyykoSerhiy** [comment]
Hm. That's odd. I'll have a look. Thx.

**Comment by ShyykoSerhiy** [comment]
Yeah, thx. My bad. Should be fixed in 0.1.2.
